processor/error/package_tests/attrs_common.go

```diff
@@ -28,20 +59,29 @@ func TestAttributesPresenceInError(t *testing.T) {
 		"errors.log.stacktrace.lineno",
 		"errors.context.request.method",
 		"errors.context.request.url",
-	)
-	condRequiredKeys := map[string]tests.Condition{
+	))
+}
+
+func condRequiredKeys(c map[string]tests.Condition) map[string]tests.Condition {
```

Comment:
why the `c` argument if you ever only call it with `nil`?

Reply:
Because I pulled this out from the distributed_tracing branch. You're right that it doesn't make sense yet, but as soon as we are allowing multiple formats to be sent to the server we will need this. So it didn't make sense to me to add methods where the signature changes with the next PR.